### ISSUE TITLE
Are secondary indices possbile in python?

### ISSUE THREAD

**Issue by fabiansinz** [datajoint/datajoint.github.io#4]
If not, clean up the example in `2015-05-05-tabledeclaration.md`.

**Comment by dimitri-yatsenko** [comment]
yes, they are.  We support indexes and unique indexes in both python and matlab.  We need to add documentation, tests, and examples.
